core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -1976,10 +1977,11 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext
 
   test("test full decommissioning flow") {
```

Comment:
1. After changing the default value `0`, please recover this test case.
2. Add a new test case with test prefix `SPARK-40979: ` with `SCHEDULER_MAX_RETAINED_REMOVED_EXECUTORS=1`.